### PR TITLE
Cloud integration update

### LINKED ISSUE #16: Setup Cloud SQL service
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### LINKED ISSUE #17: Connect Django and Cloud SQL
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### LINKED ISSUE #30: Set up Django "stable" endpoint on Google Cloud API
# Plan/Proposed Change
Deploy a Django API container on App Engine
Make sure we can talk to it from local machine

### LINKED ISSUE #31: Set up Continuous Deployment of Django API
# Plan/Proposed Change
Create a trigger where our App Engine Django container is redeployed after back end code is merged into master

### PR DESCRIPTION
**Pull request by robalty**
Added static files for gcloud deployment, plus app.yaml and main.py for the same. Edited settings.py to point to the Gcloud postgresql instance.
I'll leave the app alive online for now, although having the API just publicly available isn't ideal. I'll do some research into how we can make that more secure.

### CODE REVIEW

**Comment by SimonBartonPSU** [comment]
Awesome work, although I don't think I follow what's going on entirely. I get changing the settings to point to a postgres instance.

But how did the static files get generated?

And how does this get deployed? `gsutil` command that references GitHub? Might help once I can see the gcloud project

**Comment by robalty** [comment]
@SimonBartonPSU The static files are actually generated by manage.py - it's just making fixed pathways to files that gcloud needs. It's just bookkeeping for the gcloud instance. Deployment is done with the Gcloud SDK - from the folder with app.yaml you just run "gcloud app deploy" to deploy. Since deployment is already done though, and it's online already, you can just run the cloud sql proxy and run manage.py locally.

**Comment by SimonBartonPSU** [comment]
Groovy roovy, these seems even easier than the app engine

**Review by SimonBartonPSU — DISMISSED** [review]

File: django/todoproj/settings.py [hunk 1]
```diff
@@ -25,84 +25,105 @@
...
-        'USER': 'postgres',
-        'HOST': 'db',
-        'PORT': 5432,
+if os.getenv('GAE_APPLICATION', None):
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'HOST': '/cloudsql/cellular-virtue-277000:us-central1:capstone-covid',
+            'USER': 'postgres',
+            'PASSWORD': 'testpassword',
```
Comment (SimonBartonPSU) [inline django/todoproj/settings.py:83]: we may want to find a way to conceal our password and not publish it

unless this is a placeholder in which case I have derped

**Pull request closed without merging**